Dockerfile

```diff
@@ -1,16 +1,16 @@
 # syntax=docker/dockerfile:experimental
 
-# Build stage: Install ruby dependencies
+# Build stage: Install python dependencies
 # ===
-FROM ruby:2.5 AS build-site
-WORKDIR /srv
-ADD . .
-RUN bundle install
-RUN bundle exec jekyll build
+FROM ubuntu:focal AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+COPY requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
 
 # Build stage: Install yarn dependencies
 # ===
-FROM node:12-slim AS yarn-dependencies
+FROM node:13-slim AS yarn-dependencies
```

Comment:
was there a discussion about upgrading to node 13?

Reply:
no, but is there a need for one? we already use 13 for [charmhub](https://github.com/canonical-web-and-design/charmhub.io/blob/master/Dockerfile)

Reply:
Reverted to 12